### PR TITLE
Created BetterDiscord presence

### PR DESCRIPTION
**Pull request by Faelayis**
- Added setting buttons
- Added setting Show timestamps

<details>
<summary>
Preview
</summary>
<br>

imageimageimageimageimageimageimage

</details>

### CODE REVIEW

File: websites/B/BetterDiscord/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "tags": [
+    "betterdiscord",
+    "Themes",
+    "plugin"
+  ],
+  "url": "betterdiscord.app",
+  "description": {
+    "en": "BetterDiscord extends the functionality of DiscordApp by enhancing it with new features."
+  },
+  "iframe": false,
```
Comment (Bas950) [inline websites/B/BetterDiscord/dist/metadata.json:21]:
```suggestion
```

File: websites/B/BetterDiscord/presence.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
+const presence = new Presence({
+    clientId: "872429725090447360"
+  }),
+
+  // Const thing
```
Comment (Bas950) [inline websites/B/BetterDiscord/presence.ts:5]:
```suggestion
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Comment by Isa27-0** [comment]
Can the site be accepted by PreMiD? Because the BD is against Discord rules (TOS)

**Comment by Faelayis** [comment]
> Can the site be accepted by PreMiD? Because the BD is against Discord rules (TOS), and there are even warnings that the PreMiD application does not work on modifiers, on the Discord server.

oh yeah I didn't think about that🤔

**Comment by Slowlife01** [comment]
With the current guidelines it's totally fine

**Pull request merged**